src/javascript/app_2/App/app.js

```diff
@@ -4,7 +4,7 @@ import React                         from 'react';
 import { render }                    from 'react-dom';
 import { BrowserRouter as Router }   from 'react-router-dom';
 import Footer                        from '../App/Containers/Layout/footer.jsx';
-import Header                        from './Components/Layout/header.jsx';
+import Header                        from '../App/Containers/Layout/header.jsx';
```

Comment:
@kellybinary 
`Header` and `Footer` are accessiable to be imported from `./Containers/Layout/{headr|footer}` instead of `../App/Container/...`